src/internal/packager/deploy.go

```diff
@@ -44,6 +45,47 @@ func Deploy() {
 		spinner.Fatalf(nil, "Unable to find the package on the local system, expected package at %s", config.DeployOptions.PackagePath)
 	}
 
+	// If packagePath has partial in the name, we need to combine the partials into a single package
+	if strings.Contains(config.DeployOptions.PackagePath, "-000.partial-tar") {
```

Comment:
I am more of a fan of following the convention to chain filenames here, like `.tar.gz` and doing something like `.tar.part000` since underneath the process of splitting it it is still a .tar or .zst file.

Reply:
(that also prevents other tooling from thinking it can do something with just that one file alone)

Reply:
that's a really good point, I'm down to change that